app/controllers/user_sessions_controller.rb

```diff
@@ -34,7 +35,7 @@ def create
             if user.updated_at > 1.week.ago
               # we sent out a generated password and they're using it
               # log them in
-              @current_user = UserSession.create(user, params[:remember_me]).record
+              @current_user = UserSession.create(user, user_sesssion_params['remember_me']).record
```

Comment:
Travis helpfully pointed out there's an extra `s` in the middle of `sessions` here -- can you also change to double quotes when you fix that? Then we should be ready to go 🤞